ODIN_II/SRC/read_blif.cpp

```diff
@@ -640,7 +650,9 @@ void create_hard_block_nodes(hard_block_models *models, FILE *file, Hashtable *o
   	free_hard_block_ports(ports);
   	mapping_index->destroy_free_items();
 	delete mapping_index;
-  	vtr::free(mappings);
+	for (i = 0; i < count; i++)
```

Comment:
Remove these line (mapping is set to NULL, you can't access anything inside it. This is why I've made you free above and set to NULL)